nano/node/json_handler.cpp

```diff
@@ -2069,19 +2069,35 @@ void nano::json_handler::database_txn_tracker ()
 void nano::json_handler::delegators ()
 {
 	auto account (account_impl ());
+	auto count (count_optional_impl ());
+	auto threshold (threshold_optional_impl ());
+	auto head_str (request.get_optional<std::string> ("head"));
```

Comment:
As for name I would probably prefer "start" or may be even "start_account" to differentiate it from head block & "head" in history. Similar name can be used later for "frontiers" & "ledger" I guess, because it's not really "account" we use in wallet actions or account_info